# DP: Fix build of the D frontend on the Hurd and KFreeBSD.

--- a/src/gcc/d/dmd/root/object.h
+++ b/src/gcc/d/dmd/root/object.h
@@ -6,7 +6,7 @@
  * https://github.com/dlang/dmd/blob/master/src/root/object.h
  */
 
-#define POSIX (__linux__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
+#define POSIX (__linux__ || __GLIBC__ || __gnu_hurd__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
 
 #pragma once
 
